### PR TITLE
Update doc about standalone mode

### PR DESCRIPTION
**Pull request by jiangpengcheng**
master issue: #38

### CODE REVIEW

**Review by freeznet — APPROVED** [review]
@Huanli-Meng could you please help to review this PR when you have time? thanks.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,30 @@ functionsWorkerServiceNarPackage: /YOUR-NAR-PATH/mesh-worker-service-1.0-SNAPSHO
...
+```
+Replace the `YOUR-NAR-PATH` variable with your real path.
+
+And update the zookeeper and broker url in the `functions_worker.yml`
+
+```
+docker run -td --name worker -p 6750:6750 -v /YOUR-CONF/functions_worker.yml:/pulsar/conf/functions_worker.yml -v /YOUR-NAR-PATH/:/ANY-PATH apachepulsar/pulsar:2.9.2 bin/pulsar functions-worker
+```
+
+While worker service is running in standalone mode, you should start a pulsar proxy, see [configure-proxies](https://pulsar.apache.org/docs/next/functions-worker-run-separately#configure-proxies-for-standalone-function-workers)
```
Comment (Huanli-Meng) [inline README.md:37]:
```suggestion
To run the Function Mesh Worker service in standalone mode, you should start a Pulsar proxy. For details, see [configure-proxies](https://pulsar.apache.org/docs/next/functions-worker-run-separately#configure-proxies-for-standalone-function-workers).
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,30 @@ functionsWorkerServiceNarPackage: /YOUR-NAR-PATH/mesh-worker-service-1.0-SNAPSHO
...
+
+
+#### Run in k8s
+
+It's similar with running in docker, there is a demo [yaml](./examples/standalone.yaml), please update it with your configuration and apply it.
```
Comment (Huanli-Meng) [inline README.md:42]:
```suggestion
For details, see a sample [YAML file](./examples/standalone.yaml). You can update the configuration based on your requirement and apply it.
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave my comments. PTAL

File: README.md [hunk 3]
```diff
@@ -17,6 +17,30 @@ functionsWorkerServiceNarPackage: /YOUR-NAR-PATH/mesh-worker-service-1.0-SNAPSHO
...
 Replace the `YOUR-NAR-PATH` variable with your real path.
 
+### Run in standalone mode
+
+#### Run in docker
+
+Add the following configuration to the `functions_worker.yml` configuration file:
+
+```$xslt
+functionsWorkerServiceNarPackage: /YOUR-NAR-PATH/mesh-worker-service-1.0-SNAPSHOT.nar
```
[the remark this answers is not in the document]
Comment (jiangpengcheng) [reply README.md:27]: all versions are available

[the remark this answers is not in the document]
Comment (Huanli-Meng) [reply README.md:27]: So could we use something like `{{connector:version}}` to replace the specific version number?

[the remark this answers is not in the document]
Comment (jiangpengcheng) [reply README.md:27]: sure

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Pull request merged**